### PR TITLE
[11.0][MIG] l10n_it_account_stamp

### PR DESCRIPTION
**Pull request by SimoRubi**
Standard migration

### CODE REVIEW

**Comment by SimoRubi** [comment]
@OCA/local-italy-maintainers could you please check?
cc @agilebg

**Comment by eLBati** [comment]
@SimoRubi https://github.com/OCA/l10n-italy/pull/817 to review and include

**Comment by SimoRubi** [comment]
@eLBati, included all the recent changes of 10.0

File: l10n_it_account_stamp/views/res_config_settings_view.xml [hunk 1]
```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <record model="ir.ui.view" id="view_account_config_settings">
```
Comment (tafaRU) [inline l10n_it_account_stamp/views/res_config_settings_view.xml]: Please place `id` attribute before `model` according to https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rst#21format

Comment (SimoRubi) [reply l10n_it_account_stamp/views/res_config_settings_view.xml]: Done

Comment (tafaRU) [reply l10n_it_account_stamp/views/res_config_settings_view.xml]: Thanks!

**Comment by SimoRubi** [comment]
@OCA/local-italy-maintainers this has [enough reviews for a beta module](https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/oca_module_lifecycle_development_status.rst#alpha-and-beta-modules), could you please merge?
Thanks

**Pull request merged**